site/en/guides/build/index.md

```diff
@@ -48,32 +15,27 @@ The most common workflow is:
             $ docker pull openthread/environment:latest
             $ docker run -it --rm openthread/environment bash
 
-1.  Within your chosen environment, clone the OpenThread Git repository:
+1.  Within your chosen environment, clone the platform-specific OpenThread Git repository. Taking nrf52840 as an example:
 
-           $ git clone https://github.com/openthread/openthread
+           $ git clone https://github.com/openthread/ot-nrf528xx.git --recursive
 
 1.  From the cloned repository's root directory:
-    1.  Install the GNU toolchain and other dependencies (optional):
+    1.  Install the toolchain:
 
             $ ./script/bootstrap
 
-    1.  Set up the environment:
-
-            $ ./bootstrap
-
-    1.  Configure and build, using pre-defined platform examples with optional customization via common switches:
-        1.  Modify OpenThread compile-time constants in the selected platform's `/examples/platforms/{platform}/openthread-core-{platform}-config.h` file
-        1.  Build the configuration:
+    1.  Modify OpenThread compile-time constants in `/src/nrf52840/openthread-core-nrf52840-config.h` file.
+    1.  Build the configuration:
 
-                $ make -f examples/Makefile-{platform} <switches>
+            ./script/build nrf52840 UART_trans <platform-specific-args> <cmake-switches>
 
 1.  Flash the desired binary to the target platform. All generated binaries are
-    located in `/output/{platform}/bin`.
+    located in `/build/bin`.
 
 Specific instructions on building supported platforms with GNU Autotools can be
```

Comment:
Should this change if we're referencing the individual platform repos now?  Will the examples/platforms directory be removed at some point?

Reply:
Yes. Also, we're not using autotools anymore.